.github/workflows/testcafe.yml

```diff
@@ -21,3 +21,5 @@ jobs:
       - name: Build and run e2e tests on Chrome
         run: yarn test:ci
         working-directory: aries-site
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
```

Comment:
Should this have a comment mentioning token is defined in Percy's project settings?

Reply:
Also, I am noticing that the account is still under your name. Do we want to get that transferred to an HPE held account so that it doesn't get stuck with you?

Reply:
We can definitely do that. Which email do you think would be appropriate?

Reply:
Added Mike as an admin. I would add the hpe design system email, but it does not have an associated github account.

Reply:
> Should this have a comment mentioning token is defined in Percy's project settings?

Do you think a comment specifying where the token is defined is useful? (e.g. "Token can be found in Percy's project settings") --- Or, is that already well understood?